Tests/Functional/Controller/ImportTest.php

```diff
@@ -217,22 +217,30 @@ public function test_unassign_remote_to_local_category_without_category_id()
     public function test_unassign_remote_articles_from_local_category()
     {
         /** @var TestClient $client */
-        $client = $this->createBackendClient();
+        $client = self::createBackendClient();
 
```

Comment:
please insert fixtures first and assert before calling the controller that there are assignements
I'm pretty sure the route requires some parameters and what you are actual testing is that it doesn't fail if called without 
needed parameters but the function itself does nothing i guess